SConstruct

```diff
@@ -1021,43 +1021,47 @@ env['python_cmd_esc'] = quoted(env['python_cmd'])
 
 # Python 2 Package Settings
 cython_min_version = LooseVersion('0.23')
+numpy_min_version = LooseVersion('1.6.1')
```

Comment:
This is the version available on the apt repositories for Travis. I didn't really want to try to find sources and test for older versions, and I assume most people should have the ability to update to newer versions, if they haven't already. So, this seemed like a good minimum.